### PR TITLE
- add notify option when a connection to peripheral

### PR DESCRIPTION
**Pull request by devVadimAlbul**
Please you need to add this changed code to your branch for the possibility to set connection options to the device.

### CODE REVIEW

File: iOS/classes/BleModule.swift [hunk 1]
```diff
@@ -376,17 +376,34 @@ public class BleClientManager : NSObject {
...
+            
+            if let isConnection = options["notifyOnConnection"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnConnectionKey] = isConnection as AnyObject
+            }
+            if let isDisconnection = options["notifyOnDisconnection"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnDisconnectionKey] = isDisconnection as AnyObject
+            }
+            if let isNotification = options["notifyOnNotification"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnNotificationKey] = isNotification as AnyObject
+            }
```
Comment (Cierpliwy) [inline iOS/classes/BleModule.swift:394]: Could you please add remaining parameters with proper iOS version checks from https://developer.apple.com/documentation/corebluetooth/cbcentralmanager/peripheral_connection_options?

File: iOS/classes/BleModule.swift [hunk 2]
```diff
@@ -376,17 +376,34 @@ public class BleClientManager : NSObject {
...
+        safeConnectToDevice(deviceId, timeout: timeout,
+                            promise: SafePromise(resolve: resolve, reject: reject),
+                            options: connectOptions)
     }
 
     private func safeConnectToDevice(_ deviceId: UUID,
                                         timeout: Int?,
-                                        promise: SafePromise) {
+                                        promise: SafePromise,
+                                        options: [String: AnyObject]) {
```
Comment (Cierpliwy) [inline iOS/classes/BleModule.swift:406]: Small nitpick: There is convention in a file that last parameter is `promise: SafePromise`

**Comment by dariuszseweryn** [comment]
Hello @devVadimAlbul
Thank you for your contribution.
Do you plan to address the comments?

**Comment by dariuszseweryn** [comment]
Thank you for addressing the comments. One last thing — could you please revert all non-functional changes regarding indentation and leave only functional ones? (preferably matching the original style)

This would make both the review easier code better matching the style of the rest of the library.

I am looking forward to merge this after the adjustment. 👍

**Comment by mikolak** [comment]
Hi, @devVadimAlbul! Could you revet the whitespace changes in `BleModule.swift`?

**Comment by JamesMcIntosh** [comment]
Looks like this has been stuck for a long time, @devVadimAlbul are you willing to revert the whitespace changes?
@mikolak @dariuszseweryn if Vadim doesn't respond could one of you please take this branch over to get it merged?
Many thanks, James

**Pull request closed without merging**